core-bundle/src/InsertTag/Resolver/LegacyInsertTag.php

```diff
@@ -330,10 +329,13 @@ public function __invoke(ResolvedInsertTag $insertTag): InsertTagResult
                     case 'article_url':
                         $result = $strUrl;
                         break;
+                }
+                break;
 
-                    case 'article_title':
-                        $result = StringUtil::specialcharsAttribute($objArticle->title);
-                        break;
+            // Article title
+            case 'article_title':
+                if ($objArticle = ArticleModel::findByIdOrAlias($insertTag->getParameters()->get(0))) {
+                    $result = $objArticle->title;
```

Comment:
```suggestion
                    $result = StringUtil::specialcharsAttribute($objArticle->title);
```

Or is this no longer necessary?